packages/components/src/Banner/__snapshots__/Banner.spec.tsx.snap

```diff
@@ -67,6 +67,7 @@ exports[`<Banner /> AlertDismissable story renders snapshot 1`] = `
       fill="currentColor"
       height="28px"
       role="img"
+      style="--svg-icon-size: 28px;"
```

Comment:
Seems that Jest 24 wasn't capturing these in the snapshots. But I think it's correct to include them.